otx/core/data/adapter/segmentation_dataset_adapter.py

```diff
@@ -41,9 +42,13 @@ def get_otx_dataset(self) -> DatasetEntity:
                             datumaro_polygons = MasksToPolygons.convert_mask(ann)
                             for d_polygon in datumaro_polygons:
                                 if d_polygon.label != 0:
+                                    d_polygon.label -= 1
                                     shapes.append(self._get_polygon_entity(d_polygon, image.width, image.height))
+                                    used_labels.append(d_polygon.label)
 
                     dataset_item = DatasetItemEntity(image, self._get_ann_scene_entity(shapes), subset=subset)
                     dataset_items.append(dataset_item)
 
+        # Remove unused labels from label entities
```

Comment:
Does this comment bring any additional information? The method name looks verbose enough

Reply:
I added a detailed description inside the function, do you want to add a detailed comment on this line?

Reply:
@sovrasov , If you have other opinions, could you let me know? I'll apply it. If not, do I merge this PR?

Reply:
I actually wanted to tell that we don't need this and similar comments at all, since we have a clear function name and a docstring. Feel free to merge this.